plugins/tracers/gstsharktracer.c

```diff
@@ -34,6 +35,8 @@ struct _GstSharkTracerPrivate
   GHashTable *myhooks;
 };
 
+static unsigned int g_shark_tracer_refcount = 0;
```

Comment:
`guint`

Reply:
Also, I wonder if this should be marked `volatile` as well 🤔

Reply:
Thans, went with `volatile gint` to play nicely with atomic calls